gendoc.sh

```diff
@@ -1,16 +1,22 @@
 #!/bin/bash
 
+echo $(alias)
```

Comment:
these lines are not needed right? (lines 3 and 18)

Reply:
debugging remnants. removed